rpcs3/Emu/RSX/Common/surface_store.h

```diff
@@ -656,6 +656,7 @@ namespace rsx
 				}
 			}
 
+			rsx_log.notice("rsx::surface_cache::check_for_duplicates_fast analysed %u overlapping sections and removed %u", ::size32(sections), removed_count);
```

Comment:
maybe only log this if sections and removed are > 0

Reply:
This is intentional. The procedure isn't cheap, its good to see when it fails